pv3_rust/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+export PATH="$PATH:/home/$SUDO_USER/.cargo/bin"
+
+echo "Build packetvisor"
+rustup default stable
```

Comment:
This part is very hacky and don't really need to do
`sudo -E` can preserve environment variables
And "why not build as a user then just chown, setcap as a root"

Reply:
Okay. I removed the line both 3 and 6 and changed setcap to run as root by adding `sudo` in front of the line.

By the way, I don't know why using chown and how I should set chown?
setcap changes the own information of `pv3_rust` application as root:root after it's copied by the line 10, so that chown seems not needed

Reply:
My bad. `setcap` doesn't change ownership of the app, but `sudo cp` command does so.
but I still don't know the reason of using `chown`